classes/notifier/models/reminder/course_grade_range_model.php

```diff
@@ -58,16 +60,19 @@ public function get_user_ids_to_notify() {
         // Where users are in a specific course.
         global $DB;
 
-        $queryresults = $DB->get_records_sql('SELECT u.id
+        $gradebookroles = explode(',', $CFG->gradebookroles);
```

Comment:
Whats the reason this was extracted from the SQL? Might be slightly faster from cache, or to allow for CFG overriding?

Reply:
This was a commit by Misha

https://github.com/lsuonline/lsuce-block_quickmail/pull/82/commits/b200b7dacd0b97d3c3af45867a9e013968ce70a5